datadog.conf.example

```diff
@@ -43,6 +43,9 @@ api_key:
 # for more information
 # collect_instance_metadata: yes
 
+# use unique hostname for GCE hosts. see #1736
```

Comment:
first pass is done, see https://github.com/DataDog/dd-agent/wiki/Hostname-on-Google-Cloud-Engine